pkg/aerospike/checks.go

```diff
@@ -186,50 +184,48 @@ func DurabilityPrepare(p topology.ProbeableEndpoint) error {
 	// If the probe find a missmatch it will repush the keys
 	expectedAllPushedFlagVal := fmt.Sprintf("%s:%d", "v1", keyRange) // v1 represents the format of the data stored.
 
-	for namespace := range e.Namespaces {
-		// allPushedFlag indicate if a probe have pushed all data once
-		allPushedFlag, err := as.NewKey(namespace, e.Config.genericConfig.MonitoringSet, fmt.Sprintf("%s%s", keyPrefix, "all_pushed_flag"))
-		if err != nil {
-			return err
-		}
+	// allPushedFlag indicate if a probe have pushed all data once
+	allPushedFlag, err := as.NewKey(e.Namespace, e.Config.genericConfig.MonitoringSet, fmt.Sprintf("%s%s", keyPrefix, "all_pushed_flag"))
+	if err != nil {
+		return err
+	}
+
+	recVal, err := e.Client.Get(&policy.BasePolicy, allPushedFlag)
 
-		recVal, err := e.Client.Get(&policy.BasePolicy, allPushedFlag)
+	if err != nil && !err.Matches(as.ErrKeyNotFound.ResultCode) {
+		return err
+	}
+	// If the flag was found we skip the init as it has already been done
+	if recVal != nil && recVal.Bins["val"] == expectedAllPushedFlagVal {
+		return nil
```

Comment:
Is this change expected ? it doesn't seem related to the intent of the commit.

Reply:
it reapplies what was previously on L202